drivers/msgbox/sunxi-msgbox.c

```diff
@@ -140,6 +123,57 @@ static const struct msgbox_driver_ops sunxi_msgbox_driver_ops = {
 	.unregister_handler = sunxi_msgbox_unregister_handler,
 };
 
+static void
+sunxi_msgbox_handle_msg(struct device *dev, uint8_t chan, uint32_t msg)
+{
+	msgbox_handler handler = get_handler(dev, chan);
+
+	if (handler != NULL) {
```

Comment:
What does it mean if the handler is NULL? Can we include a debug message?

Reply:
There is a debug message right below this line.